src/features/add-ci-link.js

```diff
@@ -37,7 +37,12 @@ export default async function () {
 		} else {
 			request = fetchStatus();
 		}
-		select('.pagehead h1').append(await request);
+		
+		// Append to title (aware of forks and private repos)
+		select(`
+			.pagehead h1:not(.private) [itemprop=name],
+			.pagehead .private .Label
+		`).after(await request);
```

Comment:
If you know a shorter way to say _"append to h1, but before `span.fork-flag` if it exists"_, please suggest it.

Reply:
This is the best I can do, but neither is particularly nice:

```js
a = select('.pagehead h1 > :last-child');
a[a.matches('span.fork-flag') ? 'before' : 'after'](await request)
```

```js
insert = select('.pagehead span.fork-flag');
if(f) {
	insert.before(await request);
} else {
	select('.pagehead h1').append(await request);
}
```